### PR TITLE
Autoencoders

### PR DESCRIPTION
**Pull request by 0x00b1**

### CODE REVIEW

File: neuralcompression/models/lightning/_factorized_prior_autoencoder.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                + "_"
+                + "mse"
+                + "_"
+                + str(network_channels)
+                + "_"
+                + str(compression_channels)
+                + "_"
+                + str(distortion_trade_off).replace(".", "_")
+                + ".pth"
+            )
```
Comment (mmuckley) [inline neuralcompression/models/lightning/_factorized_prior_autoencoder.py:77]: Maybe use a `"/".join()` operation.

**Review by mmuckley — CHANGES_REQUESTED** [review]
Could we move all Lightning modules to the `projects` folder?

**Comment by 0x00b1** [comment]
@mmuckley done!

**Review by mmuckley — APPROVED** [review]
LGTM.

**Comment by 0x00b1** [comment]
Woo!

**Pull request merged**